service/start.go

```diff
@@ -68,11 +69,11 @@ func (d *Dependency) Start(networkID string) (containerServiceID string, err err
 		Image:   d.Image,
 		Args:    d.Args,
 		Command: d.Command,
-		Env: container.MapToEnv(map[string]string{
+		Env: xos.MapToEnv(xos.MergeMapEnvs(map[string]string{
 			"MESG_TOKEN":        d.service.Hash,
 			"MESG_ENDPOINT":     endpoint,
 			"MESG_ENDPOINT_TCP": endpoint,
-		}),
+		}, d.EnvValue)),
```

Comment:
I would always override with the MESG env, what happen if I decide to set `--env MESG_TOKEN=11234`

```go
xos.MapToEnv(xos.MergeMapEnvs(d.EnvValue, map[string]string{
			"MESG_TOKEN":        d.service.Hash,
			"MESG_ENDPOINT":     endpoint,
			"MESG_ENDPOINT_TCP": endpoint,
		}))
```

Reply:
This one is ok, the question should we ask if we need validation in this PR. By validation I mean someone can't use those envs in mesg.yml/--env

Reply:
Let's keep it simple for now: no validation.
But as anthony suggested: the "MESG" env should override the custom env